gex/README.md

```diff
@@ -28,6 +28,12 @@ Gex is a c# webserver that fetches public Beyond All Reason games, downloads the
 1. `npm run build`
 1. `dotnet run`
 1. view at https://localhost:6001/
+2. Create and permissions /replays folder
```

Comment:
while having better steps to running is great, i would prefer it be a different PR. what i'd actually like to do is use relative paths for some folders within the gex project to have it be able to just run once it's downloaded